outputs/otlp.go

```diff
@@ -0,0 +1,157 @@
+package outputs
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"hash/fnv"
+	"log"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/falcosecurity/falcosidekick/types"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+// Unit-testing helper
+var getTracerProvider = otel.GetTracerProvider
+
+// newTrace returns a new Trace object.
+func (c *Client) newTrace(falcopayload types.FalcoPayload) *trace.Span {
+	traceID, _, err := generateTraceID(falcopayload, c.Config)
+	if err != nil {
+		log.Printf("[ERROR] : Error generating trace id: %v for output fields %v", err, falcopayload.OutputFields)
+		return nil
+	}
+
+	startTime := falcopayload.Time
+	endTime := falcopayload.Time.Add(time.Millisecond * time.Duration(c.Config.OTLP.Traces.Duration))
+
+	sc := trace.SpanContext{}.WithTraceID(traceID)
+	ctx := trace.ContextWithSpanContext(context.Background(), sc)
+
+	tracer := getTracerProvider().Tracer("falco-event")
+	_, span := tracer.Start(
+		ctx,
+		falcopayload.Rule,
+		trace.WithTimestamp(startTime),
+		trace.WithSpanKind(trace.SpanKindServer))
+
+	span.SetAttributes(attribute.String("uuid", falcopayload.UUID))
+	span.SetAttributes(attribute.String("source", falcopayload.Source))
+	span.SetAttributes(attribute.String("priority", falcopayload.Priority.String()))
+	span.SetAttributes(attribute.String("rule", falcopayload.Rule))
+	span.SetAttributes(attribute.String("output", falcopayload.Output))
+	span.SetAttributes(attribute.String("hostname", falcopayload.Hostname))
+	span.SetAttributes(attribute.StringSlice("tags", falcopayload.Tags))
+	for k, v := range falcopayload.OutputFields {
+		span.SetAttributes(attribute.String(k, fmt.Sprintf("%v", v)))
+	}
+	// span.AddEvent("falco-event")
+	span.End(trace.WithTimestamp(endTime))
+
+	if c.Config.Debug {
+		log.Printf("[DEBUG] : OTLP payload generated successfully for traceid=%s", span.SpanContext().TraceID())
+	}
+
+	return &span
+}
+
+// OTLPPost generates an OTLP trace _implicitly_ via newTrace() by
+// calling OTEL SDK's tracer.Start() --> span.End(), i.e. no need to explicitly
+// do a HTTP POST
+func (c *Client) OTLPPost(falcopayload types.FalcoPayload) {
```

Comment:
```suggestion
func (c *Client) OTLPTracesPost(falcopayload types.FalcoPayload) {
```

because other OTLP implementations may be created in future